### PR TITLE
[doc] update nix usage for building docker image

### PR DESCRIPTION
**Pull request by SyedHassanUlHaq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ docker pull ghcr.io/chipsalliance/t1-$config:latest
 docker pull ghcr.io/chipsalliance/t1-blastoise:latest
 ```
 
+Or build the image using nix and load it into docker
+```bash
+nix build -L '.#t1.$config.release.docker-layers.final-image' --no-link --print-out-paths
+docker load < /path/to/t1-$config-image.tar
+```
```
Comment (Avimitin) [inline README.md:79]:
```bash
nix build -L ".#t1.$config.release.docker-layers.final-image"
docker load -i ./result/image.tar
```

**Comment by sequencer** [comment]
please squash

**Comment by SyedHassanUlHaq** [comment]
> please squash

done

**Comment by Avimitin** [comment]
image

It appears to me that you still have two commits in one branch.

**Comment by Avimitin** [comment]
Please run git rebase to clean the git commit message body.

**Pull request closed without merging**